content/docs/common-areas/varnish.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Varnish
+weight: 5
```

Comment:
Let's not have a separate page for Varnish. All this content can be a part of the `reverseproxy.md` file you submitted in #37 (and repeated here).